code/modules/spells/spell_types/wizard.dm

```diff
@@ -286,14 +287,14 @@
 		if(distfromcaster == 0)
 			if(isliving(AM))
 				var/mob/living/M = AM
-				M.Knockdown(100)
+				M.Knockdown(100, override_hardstun = 25)

```

Comment:
Please don't make this that long of a hardstun. 10 should be sufficient.

Reply:
Honestly, this only happens if the victim is on the same turf of the user.
And I think it's pretty legit for the victim to receive extra "ouch" if the caster ever manages to pull this move off.